test/test.js

```diff
@@ -97,8 +98,7 @@ describe("s3", function () {
       Prefix: remoteRoot,
       Bucket: s3Bucket,
     };
-    var deleter = client.deleteDir(s3Params);
-    deleter.on('end', function() {
+    client.createBucket(s3Bucket, function() {
```

Comment:
Would be nice if the current s3 behavior could also be preserved.  Perhaps make this option a different environment variable or add a variable that changes the mode of the tests to be either local or against S3 - depending on the variable would change the behavior.